vizro-ai/src/vizro_ai/_vizro_ai.py

```diff
@@ -136,75 +131,3 @@ def plot(
         # TODO Tentative for integration test
         if self._return_all_text:
             return output_dict
-
-
-def _debug_helper(
-    code_string: str, max_debug_retry: int, fix_chain: Callable, df: pd.DataFrame = None
-) -> Dict[bool, str]:
-    """Debugging helper."""
-    # TODO plug logic back into component
```

Comment:
I'll leave it up to you whether you want to change this, but if I would join the project I would have no idea what this TODO is about 😄 I think adding more comments on TODOs or explanatory comments are always nice even if it's lengthy.

Reply:
This is not applicable anymore, let me clean up the todos here